config/dev.exs

```diff
@@ -7,7 +7,7 @@ use Mix.Config
 # watchers to your application. For example, we use it
 # with brunch.io to recompile .js and .css sources.
 config :designator, Designator.Endpoint,
-  http: [port: 4000],
+  http: [port: {:system, "PORT"}],
```

Comment:
What happens if `PORT` isn't defined? Can we add a default?

Reply:
looks like that's not supported as this is a runtime vs build time issue, https://elixirforum.com/t/what-is-the-difference-between-using-system-port-and-system-get-env-port-in-deployment/1975/47

I can add a sensible default in the start script?

Reply:
added